app/models/emoji_set_site_setting.rb

```diff
@@ -9,12 +9,15 @@ def self.valid_value?(val)
 
   def self.values
     @values ||= [
-      { name: "emoji_set.apple_international", value: "apple" },
+      { name: "emoji_set.openmoji", value: "openmoji" },
+      { name: "emoji_set.standard", value: "unicode" },
+      { name: "emoji_set.fluentui", value: "fluentui" },
+      { name: "emoji_set.apple_international", value: "unicode" },
       { name: "emoji_set.google", value: "google" },
       { name: "emoji_set.twitter", value: "twitter" },
       { name: "emoji_set.win10", value: "win10" },
       { name: "emoji_set.google_classic", value: "google_classic" },
-      { name: "emoji_set.facebook_messenger", value: "facebook_messenger" },
+      { name: "emoji_set.facebook_messenger", value: "unicode" },
```

Comment:
Is it safe to rename the values of the deprecated emoji sets? What happens if I previously used "Apple/International" (`apple`) when I go to site settings? The site setting value in the DB will still be `apple`, not `unicode`. Also, won't the UI be confused when multiple options have the same value?

Reply:
Hmm that's a mistake I didn't intend to modify this but just the names indeed

Reply:
Hmm yes I had this in mind, but I mixed two ideas into one. I think you are right, I will just change the names, and old values will be symlinked anyways.

Reply:
Ok I pushed what I initially had in mind